aplus_setup.py

```diff
@@ -44,6 +44,7 @@ def setup(app):
     app.add_config_value('course_head_urls', None, 'html')
     app.add_config_value('bootstrap_styled_topic_classes', 'dl-horizontal topic', 'html')
     app.add_config_value('acos_submit_base_url', 'http://172.21.0.2:3000', 'html')
+    app.add_config_value('show_correct_once', False, 'html')
```

Comment:
Is the config value really needed at all? If the MOOC-grader does not show the model solution by default, then the rst-tools does not need to write any default value for the config key.